pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -952,16 +1060,22 @@ func (gce *GCECloud) createForwardingRule(name, serviceName, region, ipAddress s
 	return nil
 }
 
-func (gce *GCECloud) createTargetPool(name, serviceName, region string, hosts []*gceInstance, affinityType api.ServiceAffinity) error {
+func (gce *GCECloud) createTargetPool(name, serviceName, region string, hosts []*gceInstance, affinityType api.ServiceAffinity, hc []*compute.HttpHealthCheck) error {
 	var instances []string
 	for _, host := range hosts {
 		instances = append(instances, makeHostURL(gce.projectID, host.Zone, host.Name))
 	}
+	hcLinks := []string{}
+	for _, c := range hc {
+		hcLinks = append(hcLinks, c.SelfLink)
+	}
+	glog.Infof("creating targetpool %v with %d health checks", name, len(hcLinks))
```

Comment:
a service of type=lb has 1 targetpool and so the length of this list will be == 1 right? suggest a comment stating the same to avoid confusion